content/applications/websites.rst

```diff
@@ -9,4 +9,5 @@ Websites
 
     websites/website
     websites/ecommerce
+    websites/forum
```

Comment:
not related to this but I would add a link to this new page from here: https://www.odoo.com/documentation/master/applications/services/helpdesk/overview/forum_and_elearning.html